### PR TITLE
Fix matlab & jabref

### LINKED ISSUE #179: matlab icon fix
Hi, 

fix.sh does not work for matlab and jabref (new icon).  

Matlab: matlab.desktop
Icon = /usr/share/icons/hicolor/48x48/apps/matlab.png (as appears in the .desktop file)

JabRef: jabref.desktop
Icon = JabRef-icon-48  (as appears in the .desktop file)

Hope that's enough information. Cheers! ^^

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**
Fix #179

### CODE REVIEW

File: tofix.csv [hunk 1]
```diff
@@ -133,6 +133,7 @@ Intel Graphics Installer,intel-linux-graphics-installer,/usr/share/intel-linux-g
 Intellij IDE,intellij-idea,hardcoded,idea
 Intellij IDE,jetbrains-idea,hardcoded,idea
 Intellij IDE,jetbrains-idea-ce,hardcoded,idea
+JabRef,jabref,JabRef-icon-48,jabref
```
Comment (Foggalong) [inline tofix.csv]: This isn't hardcoded?

Comment (bil-elmoussaoui) [inline tofix.csv]: Yes, But Numix is using Jabref as an icon name ;)

Comment (Foggalong) [inline tofix.csv]: If that's the case then the solution is to add a symlink in Numix not a fix here since it isn't hardcoded

**Comment by bil-elmoussaoui** [comment]
I removed Jabref from this commit

**Comment by palob** [comment]
Why did you remove JabRef?

**Comment by bil-elmoussaoui** [comment]
Because JabRef wasn't hardcoded, you can just created a symlink from jabref to JabRef-icon-48

**Comment by palob** [comment]
Oops, didn't watch closely :)

**Pull request merged**